client/landing/subscriptions/components/site-subscription-page/site-subscription-details.tsx

```diff
@@ -3,6 +3,7 @@ import { useLocale } from '@automattic/i18n-utils';
 import { Button } from '@wordpress/components';
 import { useTranslate, numberFormat } from 'i18n-calypso';
 import { useEffect, useState, useMemo } from 'react';
+import { useLocation } from 'react-router';
```

Comment:
Try using `getQueryArgs()`.
```suggestion
import { getQueryArgs } from 'calypso/lib/query-args';`
```